templates/projects/empty-server/files/server/middleware.development.json

```diff
@@ -0,0 +1,9 @@
+{
+  "final:after": {
+    "loopback#strongErrorHandler": {
```

Comment:
Please don't add new middleware to `loopback` namespace, that's an antipattern we have preserved only to simplify migration from LB 1.x (express 3.x) to LB 2.x (express 4.x). In general, people should install `strong-error-handler` directly.